package.json

```diff
@@ -36,16 +36,20 @@
     "test": "make test",
     "webpack": "webpack"
   },
+  "engines": {
+    "node": ">=10.13.0 <=16",
```

Comment:
why are we supporting so many node versions? wouldn't it be better to just lock it into newer node LTS?

Reply:
I'm not sure about the best practices for an SDK and related things.
In my understanding - if somebody includes the SDK code to his own project - the more node versions are supported, it's more likely the project's node can be used.
However when thinking again - the SDK users have to use the compiled SDK, right?
In that case it's definitely better to lock to the exact version.

Reply:
Merged